webapp/src/components/emptyCenterPanel.tsx

```diff
@@ -1,34 +1,170 @@
-            <div className='WorkspaceInfo'>
-                <FormattedMessage
-                    id='EmptyCenterPanel.workspace'
-                    defaultMessage='This is the workspace for:'
+        <div
+            onClick={onClick}
+            className='button'
+        >
+            <i>{buttonIcon}</i>
+            <span className='button-title'>{title}</span>
+            {!readonly &&
+                <BoardTemplateMenuItemOptionsButton
+                    showBoard={showBoard!}
+                    boardTemplate={boardTemplate!}
                 />
-                <b>
-                    {workspace.title}
-                </b>
-            </div>
             }
-            <div className='Hint'>
+        </div>
+    )
+})
+
+const EmptyCenterPanel = React.memo(() => {
+    const workspace = useAppSelector(getCurrentWorkspace)
+    const templates = useAppSelector(getSortedTemplates)
+    const globalTemplates = useAppSelector<Board[]>(getGlobalTemplates)
+    const history = useHistory()
+    const dispatch = useAppDispatch()
+    const intl = useIntl()
+    const match = useRouteMatch<{boardId: string, viewId?: string}>()
+
+    useEffect(() => {
+        if (octoClient.workspaceId !== '0' && globalTemplates.length === 0) {
+            dispatch(fetchGlobalTemplates())
+        }
+    }, [octoClient.workspaceId])
+
+    const showBoard = useCallback((boardId) => {
+        const params = {...match.params, boardId: boardId || ''}
+        delete params.viewId
+        const newPath = generatePath(match.path, params)
+        history.push(newPath)
+    }, [match, history])
+
+    const newTemplateClicked = () => addBoardTemplateClicked(showBoard, intl)
+    const emptyBoardClicked = () => addBoardClicked(showBoard, intl)
+
+    let contentDisplay = (
+        <div className='Hint'>
+            <FormattedMessage
+                id='EmptyCenterPanel.no-content'
+                defaultMessage='Add or select a board from the sidebar to get started.'
+            />
+        </div>
+    )
+
+    if (Utils.isFocalboardPlugin()) {
+        contentDisplay = (
+            <div className='content'>
+                <span className='title'>
+                    <FormattedMessage
+                        id='EmptyCenterPanel.plugin.no-content-title'
+                        defaultMessage='Create a Board in {workspaceName}'
+                        values={{workspaceName: workspace?.title}}
+                    />
+                </span>
+                <span className='description'>
+                    <FormattedMessage
+                        id='EmptyCenterPanel.plugin.no-content-description'
+                        defaultMessage='Add a board to the sidebar using any of the templates defined below or start from scratch.{lineBreak} Members of "{workspaceName}" will have access to boards created here.'
+                        values={{
+                            workspaceName: <b>{workspace?.title}</b>,
```

Comment:
A template with no name shows up like this-

<img width="130" alt="Screenshot 2021-10-04 at 1 06 51 PM" src="https://user-images.githubusercontent.com/18575143/135811555-4f346909-3b71-4851-a23e-5d98eb2e431a.png">

Can we add a default text of `Untitled Template` or something similar, same as what we do for boards-

<img width="365" alt="Screenshot 2021-10-04 at 1 07 39 PM" src="https://user-images.githubusercontent.com/18575143/135811681-ac6ac210-147c-4958-8f6c-bf60282e48d8.png">

Reply:
Done! Thanks